console-src/src/components/ClientForm/ClientForm.tsx

```diff
@@ -66,17 +66,26 @@ type ClientFormProps = {
   isLoading?: boolean;
   buttonLabel: string;
   history: History;
-  client?: OAuthClient;
+  client?: Schema.Client;
 };
 
+type AuthorizedScopes =
+  | "OFFLINE"
+  | "ACCOUNTS"
+  | "USERS"
+  | "TRANSACTIONS"
+  | "TRANSFERS"
+  | "SUBSCRIPTIONS"
+  | "STATEMENTS";
+
```

Comment:
I think we can remove some duplicated code here with:
```
type AuthorizedScopes = Exclude<ScopeType, ScopeType.Clients | ScopeType.Admin>

type Scopes = {
  [index in AuthorizedScopes]: boolean;
}
```

Reply:
Thank you 🙏I was looking for exactly something like that 🙂 Didn't know about this `Exclude` yet.

I have updated this here: 449b560d42a4e603e3a92c8031a1c294f56832a2